modules/math/shared/src/main/scala/gsp/math/Place.scala

```diff
@@ -5,12 +5,23 @@ package gsp.math
 
 import cats.Eq
 import cats.Show
+import coulomb.Quantity
+import coulomb.si.Meter
+import eu.timepit.refined.types.numeric.NonNegInt
 import monocle.Lens
 import monocle.macros.GenLens
 import java.time.ZoneId
 
 /** A point on Earth, given latitude, longitude and altitude in m above sea level. */
-final case class Place(latitude: Lat, longitude: Lon, altitude: Double, zoneId: ZoneId)
+final case class Place(
+  latitude:  Lat,
+  longitude: Lon,
+  altitude:  Quantity[NonNegInt, Meter],
```

Comment:
What do you think of using alias/newtype for `Quantities`? eg in this case `type Altitude = Quantity[NonNegInt, Meter]`.

Reply:
We have been back and forth with type aliases. I thought about adding one here
but `Altitude` as a concept doesn't necessarily means `meters`.
I recall @tpolecat was not a big fan of type aliases but I don't recall the exact reasons
Maybe it applied more to nested types

Reply:
Yeah I don't think an alias would be useful here.

Reply:
Does coulomb allow us to express the concept of length independently of what units are used?

Reply:
No, unlike squants there is no type for `Length` or `Velocity`. I was expecting that but there are only `Quantity`s